Update Version in WordPress doc for 5.5.2 and 5.6

annezazu opened the pull request:
An additional update to cover 5.5.2 and 5.6. I made a slight change to what was shared previously for 5.6 to remove the patch release mention (8.5.1) since those releases are for bug fixes that are cherry picked into the prior release.

annezazu committed:
Update Version in WordPress for 5.5.2 and 5.6

An additional update to cover 5.5.2 and 5.6. I made a slight change to what was shared previously for 5.6 to remove the patch release mention (8.5.1) since those releases are for bug fixes that are cherry picked into the prior release.

talldan commented on docs/contributors/versions-in-wordpress.md:
```diff
@@ -6,7 +6,8 @@ If anything looks incorrect here, please bring it up in #core-editor in [WordPre
 | ------------------ | ----------------- |
-| 8.5.1-9.2            | 5.6             |
+| 8.6-9.2            | 5.6             |
+| 8.6-9.2            | 5.5.2             |
```
I think this should be the same as the line below for 5.5.1:
```suggestion
| 7.6-8.5            | 5.5.2             |
```

There will be a few small bug fixes cherry-picked from beyond plugin version 8.5 in 5.5.2 (in addition to the ones from 5.5.1):
https://core.trac.wordpress.org/timeline?from=2020-10-29T01%3A05%3A28Z&precision=second

annezazu replied:
Ah ha! You're spot on. I got confused by looking [at this line](https://github.com/WordPress/wordpress-develop/blob/5.5.2/package.json#L103) rather[ than this one](https://github.com/WordPress/wordpress-develop/blob/5.5.2/package.json#L102) :D

annezazu committed:
Update versions-in-wordpress.md

Added in 5.5.3 and included the correct included versions for both 5.5.3 and 5.5.2.

talldan reviewed (approved):
🚢 LGTM!
